generate_build_config.py

```diff
@@ -45,8 +46,45 @@
   permissions: '0755'
 """
 
+PRIVATE_PPA_TEMPLATE = """
+- chroot $CHROOT_ROOT apt install apt-transport-https
+- "echo 'deb {ppa_url} xenial main' | tee $CHROOT_ROOT/etc/apt/sources.list.d/builder-extra-ppa.list"
+- "chroot $CHROOT_ROOT apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys {key_id}"
+- chroot $CHROOT_ROOT apt update
```

Comment:
I don't think this ever does, but I'll switch to "apt-get update" instead to be on the safe side.